### PR TITLE
docs: Add updated whitelabel config in readme

### PR DESCRIPTION
**Pull request by spydon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,28 +11,78 @@ We use the GitHub issue tracker only for issues related to the development of Ub
...
+# - xubuntu
+flavour: <name>
+
+# (Optional) Sets the window's title (e.g. the text in alt|super + tab)
+app-name: <string>
+
+# (Optional) Overrides the theme's accent colors
+theme:
+  light:
+    accent-color: <color-hex-code> # i.e. #ff0011
```
Comment (d-loose) [inline README.md:50]: We should clarify/emphasize that the color code needs to be a string.
```yaml
accent-color: #ff0000
```
won't work as `#` denotes a comment, only
```yaml
accent-color: "#ff0000"
```
will.

File: README.md [hunk 2]
```diff
@@ -11,28 +11,78 @@ We use the GitHub issue tracker only for issues related to the development of Ub
...
+    elevated-button-color: <color-hex-code>
+    elevated-button-text-color: <color-hex-code>
+  dark:
+    accent-color: <color-hex-code> # i.e. #ff0011
+    elevated-button-color: <color-hex-code>
+    elevated-button-text-color: <color-hex-code>
```
Comment (spydon) [inline README.md:56]:
```suggestion
# (Optional) Overrides the theme's accent colors (remember the quotes)
theme:
  light:
    accent-color: <color-hex-code> # i.e. "#ff0011"
    elevated-button-color: <color-hex-code>
    elevated-button-text-color: <color-hex-code>
  dark:
    accent-color: <color-hex-code> # i.e. "#ff0011"
    elevated-button-color: <color-hex-code>
    elevated-button-text-color: <color-hex-code>
```

File: README.md (hunk 1 above)
Comment (spydon) [reply README.md]: Ah, copy paste mistake from the google doc, updated both now.

File: README.md [hunk 3]
```diff
@@ -11,28 +11,78 @@ We use the GitHub issue tracker only for issues related to the development of Ub
...
+# Init pages (for oem only)
+# - identity: Create the first-user account 
+# - ubuntu-pro: Enable Ubuntu Pro
+# - privacy: Enable location services
+# - timezone: Set the timezone
+# - telemetry: Enable sending telemetry
+pages:
+  <page-name>:
+  image: <image-name>
+  visible: <bool>
```
Comment (d-loose) [inline README.md:84]: We've removed the `visible` option

Comment (spydon) [inline README.md:84]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -11,28 +11,77 @@ We use the GitHub issue tracker only for issues related to the development of Ub
...
+#
+# Init pages (for oem only)
+# - identity: Create the first-user account 
+# - ubuntu-pro: Enable Ubuntu Pro
+# - privacy: Enable location services
+# - timezone: Set the timezone
+# - telemetry: Enable sending telemetry
+pages:
+  <page-name>:
+  image: <image-name>
```
Comment (spydon) [inline README.md:83]:
```suggestion
    image: <image-name>
```

File: README.md (hunk 3 above)
Comment (spydon) [reply README.md]: Good catch, fixed here and in the doc

**Review by d-loose — COMMENTED** [review]
LGTM :+1:

**Review by d-loose — APPROVED** [review]

**Pull request merged**